saga/order-service/src/main/java/io/debezium/examples/saga/order/facade/KafkaEventConsumer.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.examples.saga.order.facade;
+
+import java.io.IOException;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.persistence.OptimisticLockException;
+
+import org.eclipse.microprofile.reactive.messaging.Incoming;
+
+import io.debezium.examples.saga.order.event.CreditApprovalEvent;
+import io.debezium.examples.saga.order.event.PaymentEvent;
+import io.debezium.examples.saga.order.saga.OrderPlacementEventHandler;
+
+@ApplicationScoped
+public class KafkaEventConsumer {
+
+    private static final int RETRIES = 5;
```

Comment:
might benefit from an externalized app config property?

Reply:
For an actual implementation, yes :)